src/get_target_data.py

```diff
@@ -177,10 +188,14 @@ def set_collection_max_date(ctx, param, value):
 )
 @click.option(
     "--target-data-dir",
-    type=Path,
+    type=str,
```

Comment:
We no longer want the Github action's `create-target-data` job to write to the `target-data` directory. Thus, we modified this parameter type from Path to str so we can pass in the Github action's working directory instead.